person.rb

```diff
@@ -20,6 +22,10 @@ def can_use_services?
     is_of_age? || @parent_permission
```

Comment:
- [ ] Here you have used `is_of_age?` but you have defined `of_age?` method on line 31. kindly change this to `of_age?`.

Reply:
Thanks Mr. @Deepakdanger for your productive feedback.